dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json

```diff
@@ -15600,12 +15600,15 @@
 									"    var jsonData = pm.response.json();",
 									"    pm.expect(jsonData.entity.fileAsset).to.include(\"greatings.txt\");",
 									"    pm.expect(jsonData.entity.contentType).to.be.eql(\"FileAsset\");",
+									"    pm.expect(jsonData.entity.metaData.name).to.be.eql(\"greatings.txt\");",
```

Comment:
"greetings"

Reply:
jajaja true but it was like that, I am reusing a test